source/shared/net/nodes/troubleshoot-docker.rst

```diff
@@ -0,0 +1,72 @@
+.. _troubleshoot-node-docker:
+
+===========================================
+Troubleshoot a node on a server with Docker
+===========================================
+
+.. contents::
+   :local:
+   :backlinks: none
+
+This guide describes how to troubleshoot a node on the Concordium network running in Docker.
+
+Database invariant violation error
+==================================
+
+This error occurs due to the node running out of memory during the protocol update, which is more memory intensive than normal operation.
+
+Your node state directory should look something like this:
+
+.. code-block:: console
+
+   -rw-r--r-- 1 concordium-mainnet-node concordium-mainnet-node 3.5G Nov 19 23:32 blockstate-0.dat
+   -rw-r--r-- 1 concordium-mainnet-node concordium-mainnet-node 2.3G Dec  6 12:00 blockstate-1.dat
+   -rw-r--r-- 1 concordium-mainnet-node concordium-mainnet-node 1.3G Jan 19 08:17 blockstate-2.dat
+   drwxr-xr-x 2 concordium-mainnet-node concordium-mainnet-node 4.0K Nov 19 16:55 treestate-0
+   drwxr-xr-x 2 concordium-mainnet-node concordium-mainnet-node 4.0K Nov 19 23:32 treestate-1
+   drwxr-xr-x 2 concordium-mainnet-node concordium-mainnet-node 4.0K Dec  6 12:00 treestate-2
```

Comment:
This is how it'll look on ubuntu/debian package. I'd recommend not including users/permissions and just say
```
blockstate-0.dat
blockstate-1.dat
blockstate-2.dat
treestate-0
treestate-1
treestate-2
```
(in particular there should be matching treestate and blockstate, the number of the files will differ depending on which protocol version we are on).